### PR TITLE
DOCS-6177 added the role details for uploading a custom policy to exc…

### PR DESCRIPTION
**Pull request by priyaayyar**
…hange

### CODE REVIEW

**Review by nahuel-dallavecchia — APPROVED** [review]

File: modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 include::_attributes.adoc[]
 endif::[]
+:keywords: custom policy, upload, Exchange
 
-Exchange provides an API to publish or consume assets using a Maven client. You must have a unique artifact name and organization ID to publish your asset. +
-See the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Maven facade API] Exchange asset for more details.
+Exchange provides an API to publish or consume assets using a Maven client. You must have a unique artifact name and organization ID to publish your asset. API Manager lists custom policies published as assets of your organization.
 
-API Manager lists custom policies published as assets of your organization.
+For more information about the Exchange asset, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Maven facade API].
```
Comment (dhirsch1) [inline modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc]: So the Exchange topic is called "Exchange Maven Facade" and the API seems to be called "Exchange Maven Facade API." Your reference should match one of these - depending on whether you want to refer to the topic or the API.

**Review by dhirsch1 — APPROVED** [review]
Please see my comment.

**Review by sbehrends — APPROVED** [review]
LGTM 🚀

File: modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc]: Done! Thanks!

**Pull request merged**